src/executor/admin/ShowStatusExecutor.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+*
+* This source code is licensed under Apache 2.0 License,
+* attached with Common Clause Condition 1.0, found in the LICENSES directory.
+*/
+
+#ifndef EXECUTOR_ADMIN_SHOWSTATUSEXECUTOR_H_
+#define EXECUTOR_ADMIN_SHOWSTATUSEXECUTOR_H_
+
+#include "executor/Executor.h"
+
+namespace nebula {
+namespace graph {
+
+class ShowStatusExecutor final : public Executor {
```

Comment:
why the name is ShowStatus, not the ShowStatistics?

Reply:
Because we use the current state of the space to display the vertices and edges of the space, the number of vertices for each tag, and the number of edges for each edgetype.